labs-ios-starter/Detail/DetailViewController.swift

```diff
@@ -10,10 +10,46 @@ import UIKit
 
 class DetailViewController: UIViewController {
     
+    // MARK: - Properties
+    
+    var isFavorite: Bool = false
```

Comment:
I like this isFavorite boolean might have to tie it in with how im gonna save the favorited to an array to visualize in the favorites collection view controller.